### ISSUE TITLE
Windows installer missing on official website

### ISSUE THREAD

**Issue by rpr5x** [docker/toolbox#828]
When I visit the following link, I don't see the Docker Toolbox installer for Windows anymore: https://docs.docker.com/toolbox/toolbox_install_windows/

However, the Docker Toolbox installer for MacOS is still there:
https://docs.docker.com/toolbox/toolbox_install_mac/

Can the Windows installer be returned to the website?

**Comment by IvanDelsinne** [comment]
As a reference, current windows download link is: https://download.docker.com/win/stable/DockerToolbox.exe

Still, that's a shame to not find it anymore on the download page for windows users.

**Comment by ertemmstein** [comment]
Yes , i was searching download link  for 15 minutes between documents.
